test.go

```diff
@@ -44,6 +49,8 @@ type Test struct {
 	Middleware *Middleware
 	Request    *Request
 	Expect     *Expect
+
+	SanitizeURL SanitizeHook
```

Comment:
```suggestion
	Sanitizer SanitizeHook
```

Reply:
Great suggestion, the field has been renamed :)